app/src/main/java/com/khinthirisoe/bakingapp/ui/ingredients/IngredientsActivity.kt

```diff
@@ -0,0 +1,77 @@
+package com.khinthirisoe.bakingapp.ui.ingredients
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Recipe
+import com.khinthirisoe.bakingapp.data.model.Step
+import com.khinthirisoe.bakingapp.ui.steps.StepsActivity
+import kotlinx.android.synthetic.main.activity_ingredients.*
+
+
+class IngredientsActivity : AppCompatActivity(), StepsAdapter.StepRecyclerViewClickListener {
+
+    companion object {
+        const val EXTRA_BAKING = "extra_baking"
+    }
+
+    private var ingredientsAdapter: IngredientsAdapter? = null
+    private var stepsAdapter: StepsAdapter? = null
+
+    private lateinit var bakingName: String
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_ingredients)
+
+        val bakingRecipe = intent.getParcelableExtra<Recipe>(EXTRA_BAKING)
+        bakingName = bakingRecipe.name
+
+        setUpToolbar()
+
+        setUpView()
+
+        configureUI(bakingRecipe)
+    }
+
+    private fun setUpToolbar() {
+        if (supportActionBar != null) {
+            supportActionBar?.title = bakingName
+            supportActionBar?.setDisplayHomeAsUpEnabled(true)
+            supportActionBar?.setDisplayShowHomeEnabled(true)
+        }
+    }
+
+    private fun setUpView() {
```

Comment:
@KhinThiriSoe call this initView to make it consistent. You call it initView in other activities, remember?